I am currently get this error message. @ByteMeAsap do you have an idea what could be the cause and how to solve?

```
OwlError: An error occured in the owl lifecycle (see this Error's "cause" property)
    Error: An error occured in the owl lifecycle (see this Error's "cause" property)
        at handleError (http://oca-l10n-germany-17-0-pr200-2e7b9afa955e.runboat.odoo-community.org/web/assets/debug/web.assets_web.js:9193:35) (/web/static/lib/owl/owl.js:1633)
        at App.handleError (http://oca-l10n-germany-17-0-pr200-2e7b9afa955e.runboat.odoo-community.org/web/assets/debug/web.assets_web.js:13599:20) (/web/static/lib/owl/owl.js:6039)
        at Fiber._render (http://oca-l10n-germany-17-0-pr200-2e7b9afa955e.runboat.odoo-community.org/web/assets/debug/web.assets_web.js:9346:30) (/web/static/lib/owl/owl.js:1786)
        at Fiber.render (http://oca-l10n-germany-17-0-pr200-2e7b9afa955e.runboat.odoo-community.org/web/assets/debug/web.assets_web.js:9335:18) (/web/static/lib/owl/owl.js:1775)
        at ComponentNode.initiateRender (http://oca-l10n-germany-17-0-pr200-2e7b9afa955e.runboat.odoo-community.org/web/assets/debug/web.assets_web.js:10015:23) (/web/static/lib/owl/owl.js:2455)

Caused by: EvalError: Can not evaluate python expression: (bool(not format_base))
    Error: Name 'format_base' is not defined
    EvalError: Can not evaluate python expression: (bool(not format_base))
    Error: Name 'format_base' is not defined
        at evaluateExpr (http://oca-l10n-germany-17-0-pr200-2e7b9afa955e.runboat.odoo-community.org/web/assets/debug/web.assets_web.js:30261:15) (/web/static/src/core/py_js/py.js:45)
        at evaluateBooleanExpr (http://oca-l10n-germany-17-0-pr200-2e7b9afa955e.runboat.odoo-community.org/web/assets/debug/web.assets_web.js:30279:12) (/web/static/src/core/py_js/py.js:63)
        at Object.evalViewModifier (http://oca-l10n-germany-17-0-pr200-2e7b9afa955e.runboat.odoo-community.org/web/assets/debug/web.assets_web.js:92021:24) (/web/static/src/views/fields/x2many/x2many_field.js:197)
        at ListRenderer.evalColumnInvisible (http://oca-l10n-germany-17-0-pr200-2e7b9afa955e.runboat.odoo-community.org/web/assets/debug/web.assets_web.js:101198:27) (/web/static/src/views/list/list_renderer.js:983)
        at http://oca-l10n-germany-17-0-pr200-2e7b9afa955e.runboat.odoo-community.org/web/assets/debug/web.assets_web.js:100478:22 (/web/static/src/views/list/list_renderer.js:263)
        at Array.filter (<anonymous>)
        at ListRenderer.getActiveColumns (http://oca-l10n-germany-17-0-pr200-2e7b9afa955e.runboat.odoo-community.org/web/assets/debug/web.assets_web.js:100471:32) (/web/static/src/views/list/list_renderer.js:256)
        at ListRenderer.setup (http://oca-l10n-germany-17-0-pr200-2e7b9afa955e.runboat.odoo-community.org/web/assets/debug/web.assets_web.js:100323:27) (/web/static/src/views/list/list_renderer.js:108)
        at new ComponentNode (http://oca-l10n-germany-17-0-pr200-2e7b9afa955e.runboat.odoo-community.org/web/assets/debug/web.assets_web.js:9993:28) (/web/static/lib/owl/owl.js:2433)
        at http://oca-l10n-germany-17-0-pr200-2e7b9afa955e.runboat.odoo-community.org/web/assets/debug/web.assets_web.js:13590:28 (/web/static/lib/owl/owl.js:6030)
```

--- a/l10n_de_tax_statement/views/l10n_de_tax_statement_view.xml
+++ b/l10n_de_tax_statement/views/l10n_de_tax_statement_view.xml
@@ -115,16 +115,16 @@
                                     delete="false"
                                     limit="80"
                                 >
-                                    <field name="is_group" invisible="1" />
-                                    <field name="is_total" invisible="1" />
-                                    <field name="is_readonly" invisible="1" />
+                                    <field name="is_group" column_invisible="1" />
+                                    <field name="is_total" column_invisible="1" />
+                                    <field name="is_readonly" column_invisible="1" />
                                     <field name="code" readonly="1" />
                                     <field name="name" readonly="1" />
-                                    <field name="format_base" invisible="1" />
-                                    <field name="format_tax" invisible="1" />
+                                    <field name="format_base" column_invisible="1" />
+                                    <field name="format_tax" column_invisible="1" />
                                     <field
                                         name="base"
-                                        invisible="not format_base"
+                                        optional="show"
                                         readonly="is_readonly"
                                     />
                                     <button
@@ -136,7 +136,7 @@
                                     />
                                     <field
                                         name="tax"
-                                        invisible="not format_tax"
+                                        optional="show"
                                         readonly="is_readonly"
                                     />
                                     <button
@@ -191,7 +191,7 @@
                                     <field name="amount_tax" readonly="1" />
                                     <field
                                         name="l10n_de_tax_statement_include"
-                                        invisible="1"
+                                        column_invisible="1"
                                     />
                                     <button
                                         name="l10n_de_add_move_in_statement"

--- a/setup/l10n_de_tax_statement/odoo/addons/l10n_de_tax_statement
+++ b/setup/l10n_de_tax_statement/odoo/addons/l10n_de_tax_statement
@@ -1,1 +1,0 @@
-../../../../l10n_de_tax_statement

--- a/setup/l10n_de_tax_statement/setup.py
+++ b/setup/l10n_de_tax_statement/setup.py
@@ -1,6 +1,0 @@
-import setuptools
-
-setuptools.setup(
-    setup_requires=['setuptools-odoo'],
-    odoo_addon=True,
-)

